docs/en/setup/backend/backend-rocketmq-monitoring.md

```diff
@@ -0,0 +1,73 @@
+# RocketMQ monitoring
+
+SkyWalking leverages rocketmq-exporter for collecting metrics data from RocketMQ. It leverages OpenTelemetry
+Collector to transfer the metrics to
+[OpenTelemetry receiver](opentelemetry-receiver.md) and into the [Meter System](./../../concepts-and-designs/meter.md).
+
+## Data flow
+
+1. The `rocketmq-exporter` collects metrics data from RocketMQ, The RocketMQ version is required to be 4.3.2+.
```

Comment:
Could you add a link to the docs of rocketmq-exporter?

Reply:
> Could you add a link to the docs of rocketmq-exporter?

Done